src/entities.py

```diff
@@ -199,7 +199,7 @@ def __init__(self, long_name: str, location: Location, url_id: int, queue_status
         "Mensa Pasing",
         Location("Am Stadtpark 20, München", 48.141568, 11.451119),
         432,
-        None,
+        "https://beta.api.betterhm.app/v1/capacity/MENSA_PASING",
```

Comment:
The API is clearly marked as beta. Is this intended?

Reply:
we have some other stuff there too, but we'll change the domain soon and remove the beta.
so yes, it is intended